### PR TITLE
Added NftPendingQueue and offchain worker trigger

### LINKED ISSUE #22: NFT pending queue
### Off-chain workers initialisation

1. New NFT would be added to pending queue
2. Off-chain worker checks pending queue and if any then starts a worker (simple console log)

### PR DESCRIPTION
**Pull request by korzewski**
Closes #22

### CODE REVIEW

File: pallets/nft/src/lib.rs [hunk 1]
```diff
@@ -18,19 +20,29 @@ mod tests;
 pub type Dna = Vec<u8>;
 pub type Cid = Vec<u8>;
 
+#[derive(Encode, Decode, Clone, PartialEq, Eq, RuntimeDebug, Default)]
+pub struct PendingNft<AccountId, ClassId> {
+	account_id: AccountId,
+	class_id: ClassId,
```
Comment (nblogist) [inline pallets/nft/src/lib.rs:26]: @korzewski is this the same CID that will be generated from IPFS storage?

**Review by nblogist — APPROVED** [review]
LGTM! 
Over to @aelesbao

File: pallets/nft/src/lib.rs (hunk 1 above)
Comment (korzewski) [reply pallets/nft/src/lib.rs:26]: Nope, `ClassId` and `TokenId` are just indexes. 
`ClassId` = mould ID
`TokenID` = NFT ID

CID wold be added to `ClassData` https://github.com/anmolnetwork/anmol-node/pull/25/files/4c6d5dab6c8036633e2ac81f78fdeda180ea0c26#diff-9340a89ab12e3b4b5b5e36714cb73d3c82979532d77c70c143ab488ef416154bR35

Comment (nblogist) [reply pallets/nft/src/lib.rs:26]: Thanks for the clarification!

**Review by aelesbao — APPROVED** [review]

**Pull request merged**